From e7782c15a1cf4441c15a5e1bf003ca68a8d18e0f Mon Sep 17 00:00:00 2001
From: adamglw <53101365+adamglw@users.noreply.github.com>
Date: Wed, 30 Jun 2021 16:24:20 +0100
Subject: [PATCH] Apply manaslu terminal colors

---
 config.def.h | 57 ++++++++++++++++++++++++++--------------------------
 1 file changed, 29 insertions(+), 28 deletions(-)

diff --git a/config.def.h b/config.def.h
index 6f05dce..f4cb343 100644
--- a/config.def.h
+++ b/config.def.h
@@ -95,31 +95,32 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+  /* 8 normal colors */
+  [0] = "#262626", /* black  38-38-38    */
+  [1] = "#b78380", /* red    192-127-12  */
+  [2] = "#85b785", /* green  133-183-133 */
+  [3] = "#b6bd80", /* yellow 182-189-128 */
+  [4] = "#7694b4", /* blue   118-148-180 */
+  [5] = "#897cb8", /* purple 137-124-184 */
+  [6] = "#83b4bd", /* cyan   131-180-189 */
+  [7] = "#9e9e9e", /* white  158-158-158 */
+
+  /* 8 bright colors */
+  [8]  = "#5e5e5e", /* black  94-94-94    */
+  [9]  = "#ce8c84", /* red    206-140-132 */
+  [10] = "#92c592", /* green  146-197-146 */
+  [11] = "#c4cb8d", /* yellow 196-203-141 */
+  [12] = "#83a1c1", /* blue   131-161-193 */
+  [13] = "#9789c6", /* purple 151-137-198 */
+  [14] = "#91c1ca", /* cyan   145-193-202 */
+  [15] = "#b9b9b9", /* white  185-185-185 */
+
+  [255] = 0,
+
+  /*More colors can be added after 255 to use with DefaultXX */
+  [256] = "#262626", /* bg     38-38-38    */
+  [257] = "#9e9e9e", /* fg     158-158-158 */
+  [258] = "#b9b9b9", /* cursor 185-185-185 */
 };
 
 
@@ -127,9 +128,9 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+static unsigned int defaultcs = 258;
 static unsigned int defaultrcs = 257;
 
 /*
-- 
2.32.0

